### PR TITLE
release: v1.2.0

### PR DESCRIPTION
**Pull request by fauzanlinnas**
> This project is a forked version of the [Sendbird UI Kit](https://github.com/sendbird/sendbird-uikit-react) to match with the Ruangguru's internal requirements. Therefore, it is not yet set up to accept pull requests from external contributors. But you can always freely create a forked version from this repository to match your own requirements.

`release: v1.2.0`

- [x] PR title `release:1.2.0`
- [x] Add release 1.2.0 in `changelog`
- [x] Edit version in `package.json` to `1.2.0`
- [x] Add `release` labels in PR

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [2.0.0](Apr 20 2022)
+
+- Feature
+  - Sync with upstream sendbird-uikit-react, release v2.7.0 (#93)
```
Comment (suliskh) [inline CHANGELOG.md:6]: Changelognya ditambahkan ini yah mas:
- Bump sendibird `v3.1.4`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sendbird-uikit",
-  "version": "1.1.0",
+  "version": "2.0.0",
```
Comment (suliskh) [inline package.json]: Karena ini gak ada major changes, menurutku kita naikan minor version-nya saja yah mas, jadinya `1.2.0`

**Review by suliskh — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (fauzanlinnas) [reply package.json]: wokeeh

File: CHANGELOG.md (hunk 1 above)
Comment (fauzanlinnas) [reply CHANGELOG.md:6]: siaap

**Review by suliskh — APPROVED** [review]

**Comment by suliskh** [comment]
Great work, Mas @fauzanlinnas ! Shikatt

**Pull request merged**